aepsych/database/tables.py

```diff
@@ -61,6 +61,10 @@ class DBMasterTable(Base):
     experiment_name = Column(String(256))
     experiment_description = Column(String(2048))
     experiment_id = Column(String(10), unique=True)
+    participant_id = Column(String(50), unique=True, nullable=True)
```

Comment:
Do we want participant_id to be nullable?

Reply:
I originally wanted it that way but then I eventually wrote it so a UUID would be generated anyways so this line no longer makes sense so I will remove the nullability!